### DISCUSSION TITLE
Docs for lofty-rs

### DISCUSSION THREAD

**Discussion by mateolafalce** [Serial-ATA/lofty-rs#229] [General]
Hi!
I am interested in contributing to the documentation, are you thinking of opening an mdbook to show the entire project? or increase the official documentation in doc.rs??

**Comment by Serial-ATA** [comment]
Hello!

I'm just looking to improve the docs on docs.rs, an mdBook would probably be too much for Lofty. :)

**Comment by mateolafalce** [comment]
Thanks for the clarification!
